website/project/decorators.py

```diff
@@ -86,6 +86,12 @@ def wrapped(*args, **kwargs):
 
             _inject_nodes(kwargs)
 
+            if getattr(kwargs['node'], 'is_collection', True):
+                raise HTTPError(
+                    http.BAD_REQUEST,
```

Comment:
It'll probably be quick to change either way, though, if retractions feel significantly different from this.